apps/web/src/lib/components/ui/community-details/libraryDetails.svelte

```diff
@@ -1,31 +1,21 @@
 <script lang="ts">
 	import * as Dialog from '$lib/components/ui/dialog';
-	import { User } from 'lucide-svelte';
 	import { createEventDispatcher } from 'svelte';
-	import dayjs from 'dayjs';
-	import relativeTime from 'dayjs/plugin/relativeTime';
 	import type { Agent, Crew } from '$lib/types/models';
-
-	dayjs.extend(relativeTime);
+	import { AgentDetails, CrewDetails } from '$lib/components/ui/community-details';
 
 	export let type: 'agent' | 'crew';
-	export let displayedAgent: Agent | Crew;
+	export let displayedAgent: Agent | Crew
```

Comment:
if this can be a crew, the variable name shouldn't be "agent".
use a generic name like "displayedItem" that can be both. Or use two separate variables.